src/anim/evaluator/anim-target.js

```diff
@@ -41,6 +41,12 @@ class AnimTarget {
     get targetPath() {
         return this._targetPath;
     }
+
+    get isTransform() {
+        return (this._targetPath.substring(this._targetPath.length - 13) === 'localRotation') ||
+        (this._targetPath.substring(this._targetPath.length - 13) === 'localPosition') ||
+        (this._targetPath.substring(this._targetPath.length - 10) === 'localScale');
+    }
```

Comment:
you should evaluate this in the constructor and store it in this.isTransform property, instead of possibly executing it many times.